### PR TITLE
Ohio Scientific port

### PR DESCRIPTION
**Pull request by ivop**
As written in README.md, a wide variety of OSI systems are supported. Screen drivers for 540B color graphics card and VT100 serial system are included. Perhaps I'll craft Hazeltine 1420 and/or DEC VT52 drivers later. That's what the systems were sold with back in the day, although I suppose nobody uses that on real hardware anymore.

### CODE REVIEW

File: tools/build.py [hunk 1]
```diff
@@ -194,6 +195,30 @@ def mkimd(self, name, src: Target = None):
     )
 
 
+@Rule
+def img2os5(self, name, src: Target = None):
+    simplerule(
+        replaces=self,
+        ins=[src],
+        outs=[f"={name}.os5"],
```
Comment (davidgiven) [inline tools/build.py:203]: Using `{self.shortname}` here produces nicer results (even though the filename here doesn't really matter).

**Review by davidgiven — COMMENTED** [review]
Also, could you update the .github/workflows scripts with the new image names, please?

**Comment by davidgiven** [comment]
Excellent, thanks! The disk code looks like it was a marathon... I've left a couple of nitpicks, but other than that it looks good to go.

Is this the last 'major' 6502-based microcomputer that needs a port? If you can call the C64 version a port.

File: tools/build.py (hunk 1 above)
Comment (ivop) [reply tools/build.py:203]: I'm not sure what you mean. I can't find any reference to .shortname in the other build scripts.

**Comment by davidgiven** [comment]
Merging; thank you very much. That's a lot of disk images!

**Comment by ivop** [comment]
> Merging; thank you very much. That's a lot of disk images!

Haha, yes. And I even left out a serial system with 5.25" floppies. Also no 400 series with upgraded video to 540B from the 500 series, but with proper modification of the old 440 video board it could still use the ASCII keyboard.
 
> Is this the last 'major' 6502-based microcomputer that needs a port? If you can call the C64 version a port.

Maybe the Rockwell AIM-65 with either the RM65 floppy board (pretty rare in the wild) or the AH5050 IEC interface (there's the 1541 again :smile:).

**Pull request merged**